### DISCUSSION TITLE
Sliding Actuators from SoftRobots_v21_12

### DISCUSSION THREAD

**Discussion by bmauze** [SofaDefrost/SoftRobots#155] [Help - Programming with SoftRobots] [answered]
Hello everyone,

I used SOFA v19.06 with an old version of the SoftRobots. Currently, I am using normally the last version of sofa with the last version of the SoftRobots plugin. I was using the object SlidingActuator but it seems lacking from this version. Am I right ?
If I am looking to the old source code, could I adapt it to  python3 or the problem was more difficult  ?

Kind regards,
Benjamin Mauzé

**Comment by EulalieCoevoet** [comment, marked as answer]
Hi Benjamin, 

The SlidingActuator component is part of the plugin SoftRobots.Inverse. You can download our latest all-included binaries v22.04.01 from [here](https://project.inria.fr/softrobot/install-get-started-2/download/). It contains the plugins SoftRobots, SoftRobots.Inverse, SofaPython3, and STLIB. 

Note that the next release v22.06 should be available in few weeks. 

Best, 
Eulalie

**Reply by bmauze** [reply]
Thank Eulalie ! 

I will checked in the version v22.04.01 then.

Best,
Benjamin
